vehicle/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    'name': 'Vehicle',
+    'author': 'Solvos',
+    'category': 'Fleet',
```

Comment:
Maybe better this know category: https://github.com/odoo/odoo/blob/13.0/addons/fleet/__manifest__.py#L7